internal/rbd/nodeserver.go

```diff
@@ -67,6 +67,10 @@ const (
 	xfsReflinkNoSupport
 	xfsReflinkSupport
 
+	ext4PrezeroedUnset
```

Comment:
restart counting again, use `ext4PrezeroedUnset = itoa`

Reply:
go doesn't consider `iota` twice in the same const block :(